app/code/core/Mage/AdminNotification/Model/Inbox.php

```diff
@@ -131,7 +132,7 @@ public function parse(array $data)
     public function add($severity, $title, $description, $url = '', $isInternal = true)
     {
         if (!$this->getSeverities($severity)) {
-            Mage::throwException($this->__('Wrong message type'));
+            Mage::throwException(Mage::helper('adminnotification')->__('Wrong message type'));
```

Comment:
Code change: but consistent to other translations in this class

Reply:
Its a bug ... `__()` doesn't exit in model classes.

Reply:
Therefore I approved this PR